pkg/yurthub/cachemanager/cache_manager.go

```diff
@@ -52,12 +52,16 @@ type CacheManager interface {
 	UpdateCacheAgents(agents []string) error
 	ListCacheAgents() []string
 	CanCacheFor(req *http.Request) bool
+	UpdateKind(gvk schema.GroupVersionKind) error
+	DeleteKindFor(gvr schema.GroupVersionResource) error
+	ListCacheCRD() map[schema.GroupVersionResource]schema.GroupVersionKind
```

Comment:
It's not friendly to add func of process `gvr` or `gvk` for cacheManager, In Original design, only `serializer.go` can process `gvr` and `gvk`

Reply:
Thanks for the suggestion, the code has been updated.